tests/dotenv.rs

```diff
@@ -101,3 +101,46 @@ echo $DOTENV_KEY
     .suppress_dotenv_load_warning(false)
     .run();
 }
+
+#[test]
+fn path_not_found() {
+  Test::new()
+    .justfile("foo:\n\techo $NAME")
```

Comment:
The argument to `Test::justfile` is passed to `unindent`, so you can write it like this:

```suggestion
    .justfile("
      foo:
        echo $NAME
    ")
```